lib/matplotlib/tests/test_colors.py

```diff
@@ -1408,3 +1408,73 @@ def test_norm_deepcopy():
     norm2 = copy.deepcopy(norm)
     assert norm2._scale is None
     assert norm2.vmin == norm.vmin
+
+
+def test_norm_callback():
+    norm = mcolors.Normalize()
+    x = 0
+
+    def increment():
```

Comment:
Since this does nothing otherwise, `increment = unittest.mock.Mock(return_value=None)`

Reply:
Great suggestion, thanks!